plugins/auto-sizes/tests/test-optimization-detective.php

```diff
@@ -5,7 +5,11 @@
  * @package auto-sizes
  */
 
+require_once __DIR__ . '/../../../tests/class-optimization-detective-test-helpers.php';
```

Comment:
We could load this just once in bootstrap, instead of in every test.

Or maybe an autoloader at some point :-)

Reply:
Yeah, adding it to the bootstrap is a good idea. I recall we did have an autoloader previously but it was removed as discussed in https://github.com/WordPress/performance/pull/1262#discussion_r1621437918